packages/reaction-core/server/methods/orders.js

```diff
@@ -344,7 +345,7 @@ Meteor.methods({
       "shipping._id": shipment._id
     }, {
       $set: {
-        [`shipping.$.tracking`]: tracking
+        ["shipping.$.tracking"]: tracking
```

Comment:
The square brackets are unnecessary since this property name is not computed. Otherwise its fine.